ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -195,15 +195,19 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-private-shared-vpc-phz-sts-f28
-  cron: 41 11 10 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-canary-f28
+  cron: 45 21 6 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       EXTRACT_MANIFEST_INCLUDED: "true"
+      MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "mcpfoo", "mcp_node_num": "1", "mcp_node_label":
+        "node-role.kubernetes.io/worker="}]'
+      MCO_CONF_DAY2_CUSTOM_MCP_TO_BE_DELETED: mcpfoo
+      PAUSED_MCP_NAME: mcpfoo
```

Comment:
Sounds like we can add those newly added ENVs to the `openshift-upgrade-qe-test-canary` chain.
That way, we only need to add the ENVs once, rather than add them in each canary jobs.

Reply:
This was discussed, put theses env here can make any jobs not only use openshift-upgrade-qe-test-canary but also can use `mco-conf-day2-create-custom-mcp` and `mco-conf-day2-destroy-custom-mcp`. cc @jianlinliu

Reply:
This was discussed, put theses env here can make any jobs not only use openshift-upgrade-qe-test-canary but also can use `mco-conf-day2-create-custom-mcp` and `mco-conf-day2-destroy-custom-mcp`. cc @jianlinliu

Reply:
hi @jianlinliu Please help confirm do we need to update the env ariables? thanks

Reply:
I am okay with either.

Reply:
cc @qiliRedHat for aware, there is change that those env variables have been moved into chain openshift-upgrade-qe-test-canary